docs/guides/BUILDING.md

```diff
@@ -251,6 +251,46 @@ ninja -C out/host src/inet/tests:tests_run
 >
 > This means that the tests passed in a previous build.
 
+## Using `build_examples.py`
+
+The script `./scripts/build/build_examples.py` provides a uniform build
+interface into using `gn`, `cmake`, `ninja` and other tools as needed to compile
+various platforms.
+
+Use `./scripts/build/build_examples.py targets` to see a list of supported
+targets.
+
+Example build commands:
+
+```
+# Compiles and runs all tests on the host:
+/scripts/build/build_examples.py --target linux-x64-tests build
+
+# Compiles fuzzing tagets using libfuzzer (fuzzing requires clang)
+/scripts/build/build_examples.py --target linux-x64-tests-clang-asan-libfuzzer build
+
+# Compiles a esp32 example
+./scripts/build/build_examples.py --target esp32-m5stack-all-clusters build
+
+# Compiles a nrf example
+./scripts/build/build_examples.py --target nrf-nrf5340dk-pump build
+```
+
+### Fuzzing tests
```

Comment:
I think we should specify which sort of fuzzing, since we have at least two types of fuzzers in the tree, and this is only one of them.

Maybe "libfuzzer tests"?

Reply:
Updated to say "libfuzzer" and "unit tests".

What other fuzzers do we have? are they documented?

Reply:
I added some docs about `ossfuzz` ... at first I thought this is a new fuzzing library, but instead it looks like it is just an integration that picks up environment variables.

Reply:
> What other fuzzers do we have? are they documented?

@andy31415 There's a way to build a fuzzer version of all-clusters-app.  And yes, it's documented in `examples/all-clusters-app/linux/README.md`.